Where_Is_My_Wife/Assets/_Main/Scripts/UI/UIColors.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+public class UIColors 
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
```

Comment:
What is this?

Reply:
Unused class I planned to store the UI colors like selected button color etc. Not needed so will be deleting it.